### ISSUE TITLE
Accessibility improvements: keyboard navigation

### ISSUE THREAD

**Issue by teemukataja** [CSCfi/beacon-network-ui#28]
Currently some elements are unreachable without using the cursor. Fix all elements, such as that they are reachable with keyboard.

Things that are not reachable without a mouse (this list is not exhaustive)
* Example variant query
* Example range query
* Advanced search (and basic search) buttons
